src/ripple/app/ledger/impl/LedgerConsensusImp.cpp

```diff
@@ -640,7 +651,11 @@ void LedgerConsensusImp::timerEntry ()
         using namespace std::chrono;
         mCurrentMSeconds = duration_cast<milliseconds>
                            (steady_clock::now() - mConsensusStartTime);
-        mClosePercent = mCurrentMSeconds * 100 / mPreviousMSeconds;
+
+        mClosePercent = mCurrentMSeconds * 100 /
+            std::min (
+                mPreviousMSeconds,
+                duration_cast<milliseconds>(AV_MIN_CONSENSUS_TIME));
```

Comment:
Just wondering is `std::min()` really what you want?  As this is coded, any time `mCurrentSeconds` > 5s then `mClosePercent` will exceed 100.  If that's your intent, that's cool.  But it feels odd to take a `min` of a `MIN`